src/transformers/configuration_utils.py

```diff
@@ -152,6 +152,8 @@ class PretrainedConfig(object):
           typically for a classification task.
         - **task_specific_params** (:obj:`Dict[str, Any]`, `optional`) -- Additional keyword arguments to store for the
           current task.
+        - **problem_type** (:obj:`str`, `optional`) -- Problem type for ForSequenceClassification tasks. It can be one
+          of (None, "regression", "single_label_classification", "multi_label_classification"). Default is None.
```

Comment:
```suggestion
        - **problem_type** (:obj:`str`, `optional`) -- Problem type for :obj:`XxxForSequenceClassification` models. Can be one
          of (:obj:`"regression"`, :obj:`"single_label_classification"`, :obj:`"multi_label_classification"`).
```
For proper formatting of the strings compared to the rest of the doc. Also no need to say it defaults to None, it's implied if no default is specified (we do this consistently in the rest of the documentation).